lab8.md

```diff
@@ -0,0 +1,64 @@
+# Lab 8 - Auto-scaling in action
+
+<img src="https://github.com/openfaas/media/raw/master/OpenFaaS_Magnet_3_1_png.png" width="500px"></img>
+
+As described [here](http://docs.openfaas.com/architecture/autoscaling/) in the documentation Open FaaS ships with auto-scaling in this lab we will have a look how the auto-scaling works in action.
+
+## Pre-requisites:
+
+* At this point, it is expected you already have set up Open FaaS for Docker Swarm as described in [Lab 1](https://github.com/openfaas/workshop/blob/master/lab1.md).
+
+* As we need a way to invoke our lambda to demonstrate auto-scaling it is expected that you have access to [Curl](http://macappstore.org/curl/) or another to which allows you to do an HTTP-Request.
+
+
+## Exploring Auto-Scaling:
+
+Before dive into a hands-on session, we will have a brief look how we can configure the auto-scaling out of the box. 
+
+Out of the box Open FaaS is set up in a way that it will auto-scale based on the received `request per seconds`. If the defined threshold for the `request per seconds` is exceeded the AlertManager will fire. 
+
+> If you want the scaling happening based on CPU and Memory usage please have a brief look at [this blog](https://stefanprodan.com/2018/kubernetes-scaleway-baremetal-arm-terraform-installer/#horizontal-pod-autoscaling)
+
+For each fired alarm the auto-scaler will add 5 instances to the affected lambda. Open FaaS has two configuration options that allow to specify the starting/minimum amount of instances and also allows to ceil the maximum amount of instances:
```

Comment:
Please avoid the word "lambda" as it's a loaded term - i.e. "an AWS product". Let's use the word function?

Open FaaS should be OpenFaaS.

Reply:
instances -> replicas